src/mlmpfr_stubs.c

```diff
@@ -252,7 +252,7 @@ caml_mpfr_swap (value op1, value op2)
 CAMLprim value
 caml_mpfr_init_set_mpfr (value prec, value rnd, value op)
 {
-  CAMLparam0 ();
+  CAMLparam3 (prec,rnd,op);
```

Comment:
I try to keep the coding style as consistent as possible, please add a space after commas (same below).